linux_os/guide/system/network/network-nftables/nftables_rules_permanent/ansible/sle15.yml

```diff
@@ -0,0 +1,19 @@
+# platform = SUSE Linux Enterprise 15
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+{{% set nftables_family_names = ['bridge', 'arp', 'inet'] %}}
+{{{ ansible_instantiate_variables("var_nftable_master_config_file") }}}
+
+- name: Check the top-level configuration file exists
+  ansible.builtin.stat:
+    path: "{{ var_nftable_master_config_file }}"
+
+{{% for family in nftables_family_names %}}
```

Comment:
You could remove this jinja2 loop and use the Ansible loop though the `bridge`, `arp` and `inet` items.
It would make the resulting Playbook shorter and more readable.